pyanaconda/kickstart.py

```diff
@@ -669,9 +668,7 @@ def ksshowwarning(message, category, filename, lineno, file=None, line=None):
 
             # Set up the warnings module.
             warnings.showwarning = ksshowwarning
-
-            for category in kscategories:
-                warnings.filterwarnings(action="always", module=ksmodule, category=category)
+            warnings.filterwarnings(action="always", module=ksmodule, category=KickstartParseWarning)
```

Comment:
I think that we might be able to use `simplefilter` now. We don't need to check the module, because there is a special category for kickstart warnings.

Reply:
Yes, this seems to work.